tools/windows/get_version.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from packaging.version import Version
+import subprocess
+from typing import Literal
+
+import pytest
+import click
+
+
+Mode = Literal["python", "msi"]
+
+
+def get_version(description: str, mode: Mode) -> str:
+    # When the command returns something like v0.14
+    if "-" in description:
+        raw_version, count, commit_hash = description.split("-")
+    # Otherwise, like v0.14-1-f00b4r
+    else:
```

Comment:
```suggestion
    if "-" in description:
    	# Expected version like v0.14-1-f00b4r
        raw_version, count, commit_hash = description.split("-")
    else:
    	# Otherwise version like v0.14
```
Think these comments were reversed?

Reply:
Absolutely! Thanks!